### DISCUSSION TITLE
Making ver 1.9.1 compatible with .Net 8

### DISCUSSION THREAD

**Discussion by sirfana** [kirsan31/winforms-datavisualization#54] [General]
Any plan for making ver 1.9.1 compatible with .Net 8?

**Comment by kirsan31** [comment]
> Any plan for making ver 1.9.1 compatible with .Net 8?

Definitely, In next release. But I have no eta when it's come out, I hope not so late...

**Reply by sirfana** [reply]
OK. Looking forward to it

**Comment by kirsan31** [comment]
.NET 8 support added in 1.9.2.

**Reply by kirsan31** [reply]
Hi @sirfana I'm interested in one question - why did you ask for explicit support for net8? After all, libraries compiled for earlier versions are perfectly supported? I just want to make the package lighter and remove unnecessary explicit versions, leaving only those with different implementations. But what stops me is that for some reason you asked for support for net8 explicitly 🤔
